android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -783,27 +783,31 @@ public void onCameraTrackingDismissed() {
   }
 
   @Override
-  public void onAnnotationClick(Annotation annotation) {
+  public boolean onAnnotationClick(Annotation annotation) {
     if (annotation instanceof Symbol) {
       final SymbolController symbolController = symbols.get(String.valueOf(annotation.getId()));
       if (symbolController != null) {
         symbolController.onTap();
```

Comment:
Shouldn't `onTap` method return a boolean whether or not the event should be consume ? Like in android plugin ?

Reply:
@Ph0tonic I also thought about that, the problem is that we can have 0 to n listeners in flutter, so we'd need to decide which one makes the final decision. I haven't been able to come up with a good solution for this so far.

Reply:
For android plugin, they decided to return as soon as a listener consumes the event. In other word, the first listener in the order of registration, which consume the event prevent all other listeners to be triggered.

I think that this makes sense.

Reply:
Thinking about it again that definitely makes sense. 
It'll probably be a bit until I find time to make the change, though, so if you'd like to, feel free to implement it.

Reply:
We can tackle this as tailwork, thanks for :eyes: @Ph0tonic

Reply:
I'm not sure I'll have time to tackle this but in any case you're welcome. 👍